libwebthree/WebThree.cpp

```diff
@@ -61,7 +62,9 @@ WebThreeDirect::WebThreeDirect(
 		if (_params.sealEngineName == "Ethash")
```

Comment:
I think it would be better to use an enum as constructor argument for webthree and depending on that create the respective client.